src/Components/AddFoodForm.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { Divider, Input } from 'antd';
+
+function AddFoodForm(props) {
+  const [name, setName] = useState('');
+  const [image, setImage] = useState('');
+  const [calories, setCalories] = useState(0);
+  const [servings, setServings] = useState(0);
+
+  const handleNameInput = (e) => setName(e.target.value);
+  const handleImageInput = (e) => setImage(e.target.value);
+  const handleCaloriesInput = (e) => setCalories(e.target.value);
+  const handleServingsInput = (e) => setServings(e.target.value);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const newFood = {
+      name,
+      image,
+      calories,
+      servings,
+    };
+
+    props.addNewFood(newFood);
+    setName('');
+    setImage('');
+    setCalories(0);
+    setServings(0);
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <Divider>Add Food Entry</Divider>
+
+        <label>Name</label>
+        <Input value={name} type="text" name="name" onChange={handleNameInput} />
+
+        <label>Image</label>
+        <Input value={image} type="text" name="name" onChange={handleNameInput} />
```

Comment:
Here you're using `handleNameInput` instead of using `handleImageInput` thats why the image is not working